sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/static/build.sbt

```diff
@@ -39,6 +39,12 @@ InputKey[Unit]("html") := {
 
 TaskKey[Unit]("checkSize") := {
   val artifactSize = IO.readBytes((webpack in (Compile, fullOptJS)).value.head).length
-  val expected = 19790
-  assert(artifactSize == expected, s"expected: $expected, got: $artifactSize")
+  val sizeLow = 15000
+  val sizeHigh = 25000
```

Comment:
Bumping the webpack version changed the artifact size. I did a range check similar to the other test. Should I just put the exact value?